test/plots/rectanglePlotTests.ts

```diff
@@ -2,511 +2,486 @@
+        assert.strictEqual(entities[0].index, 0, "entity retrieved is at index 0");
+        assert.strictEqual(entities[1].index, 1, "entity retrieved is at index 1");
+        entities = plot.entitiesAt({ x: xScale.scale(4), y: yScale.scale(4) });
+        assert.lengthOf(entities, 1, "found only one entity when querying a point inside the second rectangle");
+        assert.strictEqual(entities[0].index, 1, "entity retrieved is at index 1");
+        entities = plot.entitiesAt({ x: xScale.scale(1), y: yScale.scale(1) });
+        assert.lengthOf(entities, 1, "found only one entity when querying a point inside the first rectangle");
+        assert.strictEqual(entities[0].index, 0, "entity retrieved is at index 0");
+        svg.remove();
+      });
+
+      it("retrieves the entities that intersect with the bounding box", () => {
+        plot.addDataset(new Plottable.Dataset(data));
+        plot.renderTo(svg);
+
+        let entities = plot.entitiesIn({
+          topLeft: { x: xScale.scale(1.5), y: yScale.scale(2.5) },
+          bottomRight: { x: xScale.scale(2.5), y: yScale.scale(1.5) } });
+        assert.lengthOf(entities, 2, "retrieved 2 entities intersect with the box");
+        assert.strictEqual(entities[0].index, 1, "the entity of index 1 is retrieved");
+        assert.strictEqual(entities[1].index, 2, "the entity of index 2 is retrieved");
+        svg.remove();
+      });
 
-    it("retrieves the entities that intersect with the given ranges", () => {
-      let xScale = new Plottable.Scales.Linear();
-      let yScale = new Plottable.Scales.Linear();
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dataset = new Plottable.Dataset(DATA);
-      let plot = new Plottable.Plots.Rectangle()
-        .x((d) => d.x, xScale).x2((d) => d.x2)
-        .y((d) => d.y, yScale).y2((d) => d.y2);
-      plot.addDataset(dataset).renderTo(svg);
-
-      let entities = plot.entitiesIn(
-        {min: xScale.scale(1.5), max: xScale.scale(2.5)},
-        {min: yScale.scale(2.5), max: yScale.scale(1.5)});
-      assert.lengthOf(entities, 2, "retrieved 2 entities intersect with the box");
-      assert.strictEqual(entities[0].index, 1, "the entity of index 1 is retrieved");
-      assert.strictEqual(entities[1].index, 2, "the entity of index 2 is retrieved");
-      svg.remove();
+      it("retrieves the entities that intersect with the given ranges", () => {
+        plot.addDataset(new Plottable.Dataset(data));
+        plot.renderTo(svg);
+
+        let entities = plot.entitiesIn(
+          {min: xScale.scale(1.5), max: xScale.scale(2.5)},
+          {min: yScale.scale(2.5), max: yScale.scale(1.5)});
+        assert.lengthOf(entities, 2, "retrieved 2 entities intersect with the box");
+        assert.strictEqual(entities[0].index, 1, "the entity of index 1 is retrieved");
+        assert.strictEqual(entities[1].index, 2, "the entity of index 2 is retrieved");
+        svg.remove();
+      });
```

Comment:
I would personally wrap the above into a describe block like
`describe("retrieving entities")`

How would that work?